packages/cdktf-cli/src/lib/dependencies/dependency-manager.ts

```diff
@@ -290,16 +293,21 @@ export class DependencyManager {
     return packageVersion;
   }
 
-  async addPrebuiltProvider(constraint: ProviderConstraint) {
+  async addPrebuiltProvider(
```

Comment:
We should rename this method, as it is not adding anything anymore, but rather returns a `PackageConstraint` that can be added later on.